setup.py

```diff
@@ -17,7 +17,7 @@
     license='MIT',
     packages=find_packages(exclude='tests'),
     package_data={'README': ['README.md']},
-    install_requires=['requests==2.2.1'],
+    install_requires=['requests>=2.2.1'],
```

Comment:
Actually, the best thing to do is to use inequalities for package dependencies, and equalities for app dependencies. See Srik/Jim/my thoughts on the topic here https://jira.team.affirm.com/browse/INFRA-2643